### PR TITLE
copy schema from pynwb

### PR DESCRIPTION
**Pull request by ajtritt**
Publish schema changes for 2.0 release

### CODE REVIEW

File: core/nwb.base.yaml [hunk 1]
```diff
@@ -168,9 +168,27 @@ groups:
         such as Volts. If the actual data is stored in millivolts, the field ''conversion''
         below describes how to convert the data to the specified SI unit.'
     dims:
-    - num_times
+    - - num_times
```
[the remark this answers is not in the document]
Comment (bendichter) [reply core/nwb.base.yaml]: @ajtritt

**Comment by t-b** [comment]
Just my 2 cents from the sideline:

For issues like combining multiple repositories I'm using git's submodule feature. It basically allows to include a repo in a repo and the child repo is checked out on a specifc commit. The feature is a bit controversial sometimes as it is not as straightforward as possible, but we are using it happily at e.g. https://github.com/AllenInstitute/ZeroMQ-XOP/tree/master/src.

**Comment by bendichter** [comment]
I'm in favor of this, though it would require us to change our workflow, altering the schema in the schema repo first. I think that would be fine, especially if we use master and dev branches and has pynwb reference dev

**Comment by oruebel** [comment]
We need to some changes to our workflow in the next few month as part of moving pynwb.form to a separate package and to setup the process for developing NBW:N 2.x. We can discuss the question of git submodules as part of that process.

**Review by oruebel — DISMISSED** [review]

**Pull request merged**